magicctapipe/scripts/lst1_magic/config.yaml

```diff
@@ -217,3 +217,4 @@ dl2_to_dl3:
     source_name: 'Crab'
     source_ra: null   # used when the source name is null
     source_dec: null   # used when the source name is null
+    interpolation_method: 'nearest' # 'nearest' - no interpolation (closest point taken), 'linear', 'cubic' - interpolation
```

Comment:
in the test I got better results with "nearest" than with "linear", which is quite disturbing, because it just takes the closest point without any interpolation

Reply:
Thanks, I keep the default configuration as "nearest"